packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -54,22 +54,34 @@ const AccountSignup = ({
         setPWInput(new_password);
     };
 
+    const loggedIn = !!cacheTrackEvents.parseCookies('client_information');
+
     // didMount lifecycle hook
     React.useEffect(() => {
-        trackEventWithCache({
-            name: 'ce_virtual_signup_form',
-            properties: {
-                action: 'signup_confirmed',
-                form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+        // eslint-disable-next-line no-console
+        cacheTrackEvents.track(
+            {
+                name: 'ce_virtual_signup_form',
+                properties: {
+                    action: 'signup_confirmed',
+                    form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+                    loggedIn,
+                },
             },
-        });
-        trackEventWithCache({
-            name: 'ce_virtual_signup_form',
-            properties: {
-                action: 'country_selection_screen_opened',
-                form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+            false
+        );
+
+        cacheTrackEvents.track(
+            {
+                name: 'ce_virtual_signup_form',
+                properties: {
+                    action: 'country_selection_screen_opened',
+                    form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+                    loggedIn,
+                },
             },
```

Comment:
@agrim-deriv we have the `pageLoadEvent` method which calls the cacheTrackEvents.track for tracking.We can specify the triggers an event on page load with specific page targeting

Reply:
Fixed.